src/js/components/SidebarFilter.js

```diff
@@ -145,7 +145,7 @@ class SidebarFilter extends mixin(QueryParamsMixin) {
 
       let labelClassSet = classNames(
         'side-list-item form-row-element form-element-checkbox ' +
-        'inverse row row-flex flush clickable',
+        'inverse row row-flex flex-align-items-center vertical-center flush clickable',
```

Comment:
One #nit that I know you didn't introduce but instead of having the `+` here we could just have a comma since classNames takes multiple arguments.